sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -194,6 +194,32 @@ public void can_parse_now_like_functions(String name, Boolean hasFsp, Boolean ha
     assertNotNull(parser.parse("SELECT id FROM test WHERE " + String.join(" AND ", calls)));
   }
 
+  private static Stream<Arguments> get_format_arguments() {
+    return Stream.of(
+        Arguments.of("GET_FORMAT(DATE,'USA')"),
+        Arguments.of("GET_FORMAT(DATE,'JIS')"),
+        Arguments.of("GET_FORMAT(DATE,'ISO')"),
+        Arguments.of("GET_FORMAT(DATE,'EUR')"),
+        Arguments.of("GET_FORMAT(DATE,'INTERNAL')"),
+        Arguments.of("GET_FORMAT(DATETIME,'USA')"),
+        Arguments.of("GET_FORMAT(DATETIME,'JIS')"),
+        Arguments.of("GET_FORMAT(DATETIME,'ISO')"),
+        Arguments.of("GET_FORMAT(DATETIME,'EUR')"),
+        Arguments.of("GET_FORMAT(DATETIME,'INTERNAL')"),
+        Arguments.of("GET_FORMAT(TIME,'USA')"),
+        Arguments.of("GET_FORMAT(TIME,'JIS')"),
+        Arguments.of("GET_FORMAT(TIME,'ISO')"),
+        Arguments.of("GET_FORMAT(TIME,'EUR')"),
+        Arguments.of("GET_FORMAT(TIME,'INTERNAL')")
+    );
+  }
+
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("get_format_arguments")
```

Comment:
You can reduce the list in `get_format_arguments` by making two lists - per each arg. Then combine using
```java
@MethodSource({"mSource1", "mSource2"})
```
junit should permute all combinations and test AFAIK

Reply:
I may need help with this as I could not get this to work. Trying to Google it (JUnit docs, stackoverflow, etc.), it seems like using `@MethodSource({"mSource1", "mSource2"})` it should just try use both methods as sources individually rather than trying to permute all combinations.

Reply:
Sorry, it was misleading advice. You have to make a function which permutes all arguments or permute them manually.